junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -642,6 +642,9 @@ public static String getFullyQualifiedMethodName(Class<?> clazz, String methodNa
 			classpathScanner.scanForClassesInPackage(basePackageName, classTester, classNameFilter));
 	}
 
+	/**
+	 * @see org.junit.platform.commons.support.ReflectionSupport#findNestedClasses(Class, Predicate)
+	 */
```

Comment:
I think we should replace the Javadoc with such a reference for the other methods that are now available in `ReflectionSupport`, too, shouldn't we?